airbyte-ci/connectors/base_images/base_images/python/bases.py

```diff
@@ -17,8 +17,62 @@ class AirbytePythonConnectorBaseImage(bases.AirbyteConnectorBaseImage):
 
     root_image: Final[published_image.PublishedImage] = PYTHON_3_9_18
     repository: Final[str] = "airbyte/python-connector-base"
-
     pip_cache_name: Final[str] = "pip-cache"
+    nltk_data_path: Final[str] = "/usr/share/nltk_data"
+    ntlk_data = {
+        "tokenizers": {"https://github.com/nltk/nltk_data/raw/5db857e6f7df11eabb5e5665836db9ec8df07e28/packages/tokenizers/punkt.zip"},
+        "taggers": {
+            "https://github.com/nltk/nltk_data/raw/5db857e6f7df11eabb5e5665836db9ec8df07e28/packages/taggers/averaged_perceptron_tagger.zip"
+        },
```

Comment:
> all of our connectors will start to fail to build 

Nope, as the base image is published to DockerHub, once its built successfully all the connectors can use it as a base image with a `FROM` like statement, the layers downloading the remote resources are not recomputed at connector build time.

This official [NLTK data](https://www.nltk.org/nltk_data/) is published on GitHub. We keep per commit URL to make sure we get static file content that can't change to boost build repro.
It's unlikely that Github deletes per commit files, unless NTLK explicitely asks them to unindex these URLs.

In any case: if it breaks it will break when we'll try to cut a new base image version, not a connector version.